2025-04-14T19:12:55.4869005Z ##[group]Run actions/github-script@v7
2025-04-14T19:12:55.4869325Z with:
2025-04-14T19:12:55.4869900Z   script: const verifyPullRequest = require('./scripts/github-actions/semantic-pull-request')

await verifyPullRequest({ context, core, github })

2025-04-14T19:12:55.4870679Z   github-token: ***
2025-04-14T19:12:55.4870888Z   debug: false
2025-04-14T19:12:55.4871086Z   user-agent: actions/github-script
2025-04-14T19:12:55.4871375Z   result-encoding: json
2025-04-14T19:12:55.4871581Z   retries: 0
2025-04-14T19:12:55.4871796Z   retry-exempt-status-codes: 400,401,403,404,422
2025-04-14T19:12:55.4872078Z ##[endgroup]
2025-04-14T19:12:56.4280075Z A changelog entry was not found in cli/CHANGELOG.md.
2025-04-14T19:12:56.4280588Z 
2025-04-14T19:12:56.4281002Z Please add a changelog entry that describes the changes. Include this entry under the section:
2025-04-14T19:12:56.4281447Z 
2025-04-14T19:12:56.4281542Z **Bugfixes:**
2025-04-14T19:12:56.4281682Z 
2025-04-14T19:12:56.4281981Z  - <Insert change details>. Fixed in [#29011](https://github.com/cypress-io/cypress/pull/29011).
2025-04-14T19:12:56.4282461Z 
2025-04-14T19:12:56.4308025Z ##[error]There was one or more errors when validating the changelog. See above for details.
